packages/smart-account/src/index.ts

```diff
@@ -0,0 +1,17 @@
+import { Executor__factory } from './contracts';
+import { getConfig, getExecutor, getExecutorId } from './configs';
+
+export function getSmartIds(chainId: number) {
```

Comment:
```suggestion
export function getSmartAccountIds(chainId: number) {
```
Prefer to keep the naming consistently